src/javascript/app/pages/trade/charts/highchart.ui.js

```diff
@@ -4,12 +4,14 @@ const localize = require('../../../../_common/localize').localize;
 const HighchartUI = (() => {
     let common_time_style,
         common_spot_style,
+        reset_time_style,
         txt,
         chart_options;
 
     const initLabels = () => {
         common_time_style = 'margin-bottom: 3px; margin-left: 10px; height: 0; width: 20px; border: 0; border-bottom: 2px; border-color: #e98024; display: inline-block;';
         common_spot_style = 'margin-left: 10px; display: inline-block; border-radius: 6px;';
+        reset_time_style  = 'margin-bottom: 3px; margin-left: 10px; height: 0; width: 20px; border: 0; border-bottom: 2px; border-color: #000; display: inline-block;';
```

Comment:
it looks exactly like `common_time_style` except for `border-color`, perhaps we can remove `border-color` from `common_time_style` and use that and add the `border-color` accordingly to each element using it